pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/tls-secrets.test.yaml

```
@@ -64,3 +64,18 @@ tests:
         verifyTLSSecret("sensor")
         .secrets["collector-tls"] | assertThat(. == null)
         .secrets["admission-control-tls"] | assertThat(. == null)
+    - name: "should create TLS secrets with rhacs.redhat.com/tls=true"
+      release:
+        isInstall: true
+      values:
+        ca:
+          cert: "CA cert"
+        collector:
+          serviceTLS:
+            cert: "collector cert"
+            key: "collector key"
+        createSecret: true
+        expect: |
+          assertThat(.secrets["sensor-tls"].metadata.labels["rhacs.redhat.com/tls"] == "true")
+          assertThat(.secrets["collector-tls"].metadata.labels["rhacs.redhat.com/tls"] == "true")
+          assertThat(.secrets["admission-control-tls"].metadata.labels["rhacs.redhat.com/tls"] == "true")
```

```suggestion
      expect: |
        assertThat(.secrets["sensor-tls"].metadata.labels["rhacs.redhat.com/tls"] == "true")
        assertThat(.secrets["collector-tls"].metadata.labels["rhacs.redhat.com/tls"] == "true")
        assertThat(.secrets["admission-control-tls"].metadata.labels["rhacs.redhat.com/tls"] == "true")
```

I just noticed that the indentation here is wrong, and because `expect` is under `values`, the tests are not executed.

Using the correct indentation will cause two of the assertions to fail:
* you need to put the label also in `image/templates/helm/stackrox-secured-cluster/templates/collector-secret.yaml`
* you need to add values for `admissionControl`